src/Yasumi/Provider/AbstractProvider.php

```diff
@@ -335,6 +335,6 @@ private function anotherTime(int $year, string $key): ?Holiday
         // Get calling class name
         $hReflectionClass = new \ReflectionClass(\get_class($this));
 
-        return Yasumi::create($hReflectionClass->getShortName(), $year, $this->locale)->getHoliday($key);
+        return Yasumi::create($hReflectionClass->getName(), $year, $this->locale)->getHoliday($key);
```

Comment:
The `getShortName` function was renamed to `getKey` in commit 829abcd971290a06c97a2c5e93e3bf1091cd713d. That should be the preferred function to be used here.

Reply:
In this PR I'm referring to the internal methods of the PHP reflection class object.
https://www.php.net/manual/en/reflectionclass.getname.php
https://www.php.net/manual/en/reflectionclass.getshortname.php

The renaming in the commit you mentioned does not affect this part of the project.
Sorry if I wasn't clear enough.

Reply:
Thanks for the explanation: I overlooked that it relates to the Provider classes, and mixed up the `getName` and `getShortName` functions in the `Holiday` class.